py/server/tests/test_partitioned_table.py

```diff
@@ -116,6 +117,12 @@ def test_constituents(self):
         self.assertGreater(len(constituent_tables), 0)
 
     def test_transform(self):
+        _JExecutionContext = jpy.get_type("io.deephaven.engine.context.ExecutionContext")
+        context = _JExecutionContext.newBuilder() \
+                .captureCompilerContext()         \
+                .captureQueryLibrary()            \
+                .emptyQueryScope()                \
+                .build().open()
```

Comment:
This puts too much burden on the users. The ExecutionContext should be transparent to the application code. We might need to redesign it or find a way in Python to wrap it.